index.html

```diff
@@ -49,10 +73,208 @@ <h2 class="CONNECT"><a href="#">LET'S CONNECT</a> </h2>
+                  <div class="img-1"><img class="fontimg"  width="100%"  height="100%" src="Snapshoot3.png" alt=""></div>
+          
+          
+                  <div class="tonic">
+                      <div><h1>Tonic</h1></div>
+                      <div class="flex1">
+                          <div>Caponi</div>
+                          <div class="round"></div>
+                          <div>back End</div>
+                          <div class="round"></div>
+                          <div>2015</div>
+                      </div>
+                      <div>
+                          <p>
+                              A dayly selection of privatelypersonalized reads:no accounts <br>
+                              or sign-ups required.
+                          </p>
+                      </div>
+                      <div >
+                          <ul class="languages">
+                          <li class="language">HTML</li>
+                          <li class="language">CSS</li>
+                          <li class="language">javascript</li>
+                          </ul>
+                      </div>
+                      <div><button class="see-project" type="button">See project</button></div>
+                  </div>
+                    </div>
+          
+          
+                    <div class="Works">
+                      <div class="img-1"><img class="fontimg"  width="100%"  height="100%" src="Snapshoot4.png" alt=""></div>
+              
+              
+                      <div class="tonic">
+                          <div><h1>Muti-Post <br>Stories</h1></div>
+      
+                          <div class="flex1">
+                              <div>Caponi</div>
+                              <div class="round"></div>
+                              <div>back End</div>
+                              <div class="round"></div>
+                              <div>2015</div>
+                          </div>
+      
+                          <div>
+                              <p>
+                                  A dayly selection of privatelypersonalized reads:no accounts <br>
+                                  or sign-ups required.
+                              </p>
+                          </div>
+      
+                          <div >
+                              <ul class="languages">
+                              <li class="language">HTML</li>
+                              <li class="language">CSS</li>
+                              <li class="language">javascript</li>
+                              </ul>
+                          </div>
+      
+                          <div ><button class="see-project" type="button">See project</button></div>
+                      </div>
+                        </div>
+                   
+          
+          
+           <div class="about-me" >
+          
+             <div class="first-about">
+              <div >
```

Comment:
-  Kindly make sure to use a <section> tag for the about section in order to respect the semantic HTML rules :thinking: 
- In my opinion, it would be better to style your about section as per the Figma design as it is a requirement for this project, for that you will need to inspect the figma design.
Your design:
![Screenshot from 2023-02-21 15-43-48](https://user-images.githubusercontent.com/97834160/220362633-b2b469f4-762e-4eb3-a49c-d5d9002c06aa.png)
Figma:
![Screenshot from 2023-02-21 15-46-41](https://user-images.githubusercontent.com/97834160/220362810-97f76416-1cee-4bd5-afa9-fe6ff5887a68.png)